BREEZE.rst

```diff
@@ -474,214 +617,213 @@ Run pylint checks for all files:
      ./breeze --static-check-all-files pylint
 
 
-The ``license`` check is also run via separate script and separate docker image containing
+The ``license`` check is run via a separate script and a separate docker image containing the 
 Apache RAT verification tool that checks for Apache-compatibility of licences within the codebase.
-It does not take pre-commit parameters as extra args.
+It does not take pre-commit parameters as extra arguments.
 
 .. code-block:: bash
 
      ./breeze --static-check-all-files licenses
 
-Building the documentation
---------------------------
+Running Static Code Checks from the Host
+----------------------------------------
 
-The documentation is build using ``-O``, ``--build-docs`` command:
+You can trigger the static checks from the host environment, without entering the Docker container. To do this, run the following scripts (the same is done in Travis CI):
 
-.. code-block:: bash
+* `<scripts/ci/ci_check_license.sh>`_ - checks the licenses.
+* `<scripts/ci/ci_docs.sh>`_ - checks that documentation can be built without warnings.
+* `<scripts/ci/ci_flake8.sh>`_ - runs Flake8 source code style enforcement tool.
+* `<scripts/ci/ci_lint_dockerfile.sh>`_ - runs lint checker for the Dockerfile.
+* `<scripts/ci/ci_mypy.sh>`_ - runs a check for mypy type annotation consistency.
+* `<scripts/ci/ci_pylint_main.sh>`_ - runs pylint static code checker for main files.
+* '`<scripts/ci/ci_pylint_tests.sh>`_ - runs pylint static code checker for tests.
 
-     ./breeze --build-docs
+The scripts may ask you to rebuild the images, if needed.
 
-Results of the build can be found in ``docs/_build`` folder. Often errors during documentation generation
-come from the docstrings of auto-api generated classes. During the docs building auto-api generated
-files are stored in ``docs/_api`` folder - so that in case of problems with documentation you can
-find where the problems with documentation originated from.
+You can force rebuilding the images by deleting the [.build](./build) directory. This directory keeps cached
+information about the images already built and you can safely delete it if you want to start from scratch.
 
-Running tests directly from host
---------------------------------
+After documentation is built, the html results are available in the [docs/_build/html](docs/_build/html) folder.
```

Comment:
```suggestion
After documentation is built, the HTML results are available in the [docs/_build/html](docs/_build/html) folder.
```

I think there are many more. Should I comment on each one?

Reply:
We'll review :)